src/codeunit/SpotifyService/ITICreateSpotifyPlaylist.Codeunit.al

```diff
@@ -0,0 +1,178 @@
+codeunit 50100 "ITI Create Spotify Playlist"
+{
+    trigger OnRun()
+    var
+        ArtistsIDList: List of [Text];
+        ArtistList: List of [Text];
+        ArtistJsonObjList: List of [JsonObject];
+        TracksIDString: Text;
+        PlaylistID: Text;
+        PlaylistName: Text;
+        PlaylistDesc: Text;
+    begin
+        CreatePlaylist(PlaylistID, PlaylistName, PlaylistDesc);
+        IterateArtists(ArtistList, ArtistsIDList, ArtistJsonObjList);
+        GetArtistTopTracksUris(ArtistsIDList, TracksIDString);
+        AddTracksToPlaylist(PlaylistID, TracksIDString);
+    end;
+
+    procedure Create(var ArtistList: List of [Text]; PlaylistName: Text; PlaylistDesc: Text)
```

Comment:
Tak jak pisałem w poprzednim komentarzu - parametr `ArtistList: List of [Text]` zamieniłbym parametrem `var ITIArtist: Record "ITI Artist"`